src/main/asciidoc/reference/authentication.adoc

```diff
@@ -722,6 +722,8 @@ class AppConfig extends AbstractVaultConfiguration {
 
     // …
 }
+
```

Comment:
Seems some changes made it into the documentation that shouldn't be here.

Reply:
Indeed, c&p mistake